docs/tutorials/pytorch-mnist-tutorial.txt

```diff
@@ -37,24 +35,15 @@ Here is what the skeleton of our trial class looks like:
 .. code:: python
 
     import torch.nn as nn
-    from determined.pytorch import DataLoader, PyTorchTrial
+    from determined.pytorch import DataLoader, PyTorchTrial, PyTorchTrialContext
 
     class MNISTTrial(PyTorchTrial):
-        def __init__(self, context: det.TrialContext):
-            # Initialize the trial class.
-            pass
-
-        def build_model(self):
-            # Build the model.
-            pass
-
-        def optimizer(self, model: nn.Module):
-            # Define the optimizer.
+        def __init__(self, context: PyTorchTrialContext):
+            # Initialize the trial class and wrap the models, optimizers, and LR schedulers.
             pass
 
         def train_batch(self, batch: TorchData, model: nn.Module, epoch_idx: int, batch_idx: int):
```

Comment:
blocking: when we make this public, the examples should not include a `model` argument in `train_batch`